pages/deployment/[uuid].tsx

```diff
@@ -220,6 +220,24 @@ export default function Deployment() {
                 </Alert>
               </Box>
             )}
+            {hasUploadType && initialVersionRequested?.metadata.buildOptions.uploadType === ModelUploadType.Docker && (
+              <Box>
```

Comment:
Not for this ticket, but I wonder if we can change how the alerts on these pages work. We could either use a  re-usable component, but I don't know if there might be cases where there should be multiple alerts?